depends/config.site.in

```diff
@@ -25,6 +28,10 @@ if test -z $enable_wallet && test -n "@no_wallet@"; then
   enable_wallet=no
 fi
 
+if test -z $enable_multiprocess && test -n "@multiprocess@"; then
+  enable_multiprocess=yes
+fi
+
```

Comment:
This is a very convenient feature.
But, OTOH, it is a bit confusing as `--enable-multiprocess` help string states that "default is **no**".

May I suggest to drop this addition?

As another approach, the default value of `--enable-multiprocess` could be `auto` (but not now, when it is an experimental feature).

Reply:
Good points. I think as long as this is experimental, probably best to play it safe and require it to be --enabled by hand. Arguably it already has been because of the way depends was built, but it's easy to forget when bouncing around branches.

Not a strong preference, though.